.github/CITATION.cff

```diff
@@ -0,0 +1,12 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "The FLARE Team"
+  given-names: ""
+  orcid: ""
+title: "Capa , a tool to identify program capabilities in executables and sandbox traces."
+version: 7.0.1
```

Comment:
yeah thought so , added it in case we would like to keep the version while getting cited , same for the names and such. sorry for the Bad PR lemme commit the changes